src/st/no0/e_diplocephalus.c

```diff
@@ -68,7 +68,7 @@ void EntityDiplocephalusFoot(Entity* self) {
             self->animCurFrame++;
             if (self->ext.diplocephalus.unk9E < 3) {
                 if (g_CurrentEntity->facingLeft ^
-                    (LOHU(self->ext.prim->x3) % 2)) {
+                    (self->ext.diplocephalus.parent->step % 2)) {
```

Comment:
Realised while decompiling one of the later functions I had this wrong, this now makes a lot more sense than randomly using prim